src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -606,12 +670,14 @@ internal HttpCreationPayload GetInstanceCreationLinks()
                 SendEventPostUri = instancePrefix + "/" + RaiseEventOperation + "/{eventName}?" + querySuffix,
                 TerminatePostUri = instancePrefix + "/" + TerminateOperation + "?reason={text}&" + querySuffix,
                 RewindPostUri = instancePrefix + "/" + RewindOperation + "?reason={text}&" + querySuffix,
+                PurgeInstanceHitoryByInstanceIdDeleteUri = instancePrefix + "?" + querySuffix,
+                PurgeInstanceHitoryWithFiltersDeleteUri = allInstancesPrefix + "?" + "createdTimeFrom={createdTimeFrom}&createdTimeTo={createdTimeTo}&runtimeStatus={runtimeStatus,runtimeStatus,...}&" + querySuffix,

```

Comment:
Same comment as before. I think we shouldn't include the filtered purge URL since it is not specific to the target instance.

Reply:
I will fix it. Thank you!